rancher_pr_deploy.sh

```diff
@@ -55,14 +55,18 @@ rancher() {
     $@
 }
 
+yml2json() {
+  echo "$@" | docker run -a python:alpine -c 'import sys, yaml, json; json.dump(yaml.load(sys.stdin), sys.stdout)'
+}
+
 if [ "$TRAVIS" == true ] && [ "$TRAVIS_PULL_REQUEST" != false ] ; then
 
   fold_start "Generate_answers.txt"
   # POPULATES ANSWERS FILE WITH THE DEFAULTS FROM THE RANCHER-COMPOSE FILE
   RANCHER_CATALOG_ID=$( curl -su "${RANCHER_ACCESS_KEY}:${RANCHER_SECRET_KEY}" "${RANCHER_SERVER}/v1-catalog/templates/${RANCHER_CATALOG_NAME}:${RANCHER_TEMPLATE_NAME}" | jq --raw-output '.defaultTemplateVersionId' )
   curl -su "${RANCHER_ACCESS_KEY}:${RANCHER_SECRET_KEY}" "${RANCHER_SERVER}/v1-catalog/templates/${RANCHER_CATALOG_ID}" | \
   jq --raw-output '.files["rancher-compose.yml"]' | \
-    python -c 'import sys, yaml, json; json.dump(yaml.load(sys.stdin), sys.stdout)' | \
+    yml2json | \
```

Comment:
Also, not sure how you weigh up the benefits of running a single command (python/rancher/etc) in a docker container vs installing it in Travis. I guess as you mentioned in the catchup it makes the script more self contained and able to be run in environments other than Travis?